charts/redpanda/values.go

```diff
@@ -440,29 +432,24 @@ func (s *Storage) StorageMinFreeBytes() int64 {
 }
 
 type PostInstallJob struct {
-	Resources *corev1.ResourceRequirements `json:"resources"`
-	Affinity  map[string]any               `json:"affinity"`
-
-	// Fields that are in values.yaml but not in values.schema.json.
-	Enabled         bool                   `json:"enabled"`
-	Labels          map[string]string      `json:"labels"`
-	Annotations     map[string]string      `json:"annotations"`
-	SecurityContext corev1.SecurityContext `json:"securityContext"`
+	Resources       *corev1.ResourceRequirements `json:"resources"`
+	Affinity        corev1.Affinity              `json:"affinity"`
+	Enabled         bool                         `json:"enabled"`
+	Labels          map[string]string            `json:"labels"`
+	Annotations     map[string]string            `json:"annotations"`
+	SecurityContext *corev1.SecurityContext      `json:"securityContext"`
```

Comment:
I wonder if `BackoffLimit`, `ExtraEnv` and `ExtraEnvFrom` should be added to match `PostUpgradeJob` struct.
```suggestion
	Resources       *corev1.ResourceRequirements `json:"resources"`
	Affinity        corev1.Affinity              `json:"affinity"`
	Enabled         bool                         `json:"enabled"`
	Labels          map[string]string            `json:"labels"`
	Annotations     map[string]string            `json:"annotations"`
        BackoffLimit    *int32                      `json:"backoffLimit"`
	ExtraEnv        []corev1.EnvVar             `json:"extraEnv"`
	ExtraEnvFrom    []corev1.EnvFromSource      `json:"extraEnvFrom"`
	SecurityContext *corev1.SecurityContext      `json:"securityContext"`
```